src/components/AnonymousChat/child/ChannelFollowButton.tsx

```diff
@@ -3,24 +3,31 @@ import React from 'react';
 import {Pressable, StyleSheet, Text, View} from 'react-native';
 
 import dimen from '../../../utils/dimen';
-import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
+import {COLORS} from '../../../utils/theme';
 
 interface Props {
   isFollowing: boolean;
   handleFollow?: () => void;
+  isAnonymousTab?: boolean;
 }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isAnonymousTab` has been added to the `Props` interface. This is a good change as it allows for more customization of the component based on whether the tab is anonymous or not. However, please ensure that all instances where this component is used are updated to pass this prop if necessary.

```diff
  interface Props {
    isFollowing: boolean;
    handleFollow?: () => void;
+   isAnonymousTab?: boolean;
  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->